### ISSUE TITLE
Something question about datasets processed (with CLIP-ViT/B-32)

### ISSUE THREAD

**Issue by LearningJack** [drboog/Lafite#12]
Could you help me? You provide several commonly used datasets that you have already processed (with CLIP-ViT/B-32) in datasets.json. So how I should do in my datasets and how to keep processed image order as origin unprocessed image?
Another question is why running train.py needs the parameter with val_dataset?

**Comment by drboog** [comment]
1. You can have a look at dataset_tool.py and revise some code there to maintain order, or their original names.
2. That is optional, you do not have to use that if you are running on your own dataset. It is used following previous works because they evaluate metrics on COCO by generating images with text from validation set, instead of text from training set.
